bounties/pip/computervision-recipes/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-12-09",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Contributor": {
+      "Discloser": "B3EF",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
+      "Name": "computervision-recipes",
+      "URL": "https://github.com/microsoft/computervision-recipes.git"
+    },
+    "CWEs": [
+      {
+        "ID": "94",
+        "Description": "Improper Control of Generation of Code ('Code Injection')"
+      }
+    ],
+    "CVSS": {
+      "Version": "3.1",
+      "AV": "",
+      "AC": "",
+      "PR": "",
+      "UI": "",
+      "S": "",
+      "C": "",
+      "I": "",
+      "A": "",
+      "E": "",
+      "RL": "",
+      "RC": "",
+      "Score": "8.x"
```

Comment:
@B3EF - Can you please select a specific value instead of `8.x`?

Reply:
Sure..✌️